_posts/2017-01-16-Jak-psat-php-Uloziste-kodu+databaze+frameworky.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Jak psát PHP? (3/3) - Úložiště kódu, databáze a frameworky
```

Comment:
Jop, vnímám to taky. Článek můžu pojmenovat Jak začít psát/se psaním/programovat PHP? :D
Ale ten už je na Pehapkari, tak nevím zda to psát znovu a lépe